src/__tests__/surveys.test.ts

```diff
@@ -119,674 +129,658 @@ describe('surveys', () => {
+                expect(sortedQuestions.length).toEqual(shuffledQuestions.length)
+                const sortedShuffledQuestions = shuffledQuestions.sort(function (a, b) {
+                    return a.question.localeCompare(b.question)
+                })
+                expect(sortedQuestions).toEqual(sortedShuffledQuestions)
+            })
         })
 
-        it('shuffle should preserve all elements', () => {
-            const shuffledOptions = getDisplayOrderChoices(questionWithOpenEndedChoice)
-            const sortedOptions = questionWithOpenEndedChoice.choices.sort()
-            const sortedShuffledOptions = shuffledOptions.sort()
+        describe('shuffling options', () => {
+            const questionWithoutShufflingOptions: MultipleSurveyQuestion = {
+                type: SurveyQuestionType.MultipleChoice,
+                question: "We're sorry to see you go. What's your reason for unsubscribing?",
+                choices: [
+                    'I no longer need the product',
+                    'I found a better product',
+                    'I found the product too difficult to use',
+                    'Other',
+                ],
+                hasOpenChoice: true,
+                shuffleOptions: false,
+            } as unknown as MultipleSurveyQuestion
+
+            const questionWithShufflingOptions: MultipleSurveyQuestion = {
+                type: SurveyQuestionType.MultipleChoice,
+                question: "We're sorry to see you go. What's your reason for unsubscribing?",
+                choices: [
+                    'I no longer need the product',
+                    'I found a better product',
+                    'I found the product too difficult to use',
+                    'Other',
+                ],
+                hasOpenChoice: true,
+                shuffleOptions: true,
+            } as unknown as MultipleSurveyQuestion
+
+            const questionWithOpenEndedChoice: MultipleSurveyQuestion = {
+                type: SurveyQuestionType.MultipleChoice,
+                question: "We're sorry to see you go. What's your reason for unsubscribing?",
+                choices: [
+                    'I no longer need the product',
+                    'I found a better product',
+                    'I found the product too difficult to use',
+                    'open-ended-choice',
+                ],
+                hasOpenChoice: true,
+                shuffleOptions: true,
+            } as unknown as MultipleSurveyQuestion
 
-            expect(sortedOptions).toEqual(sortedShuffledOptions)
-        })
-    })
+            it('should not shuffle if shuffleOptions is false', () => {
+                const shuffledOptions = getDisplayOrderChoices(questionWithoutShufflingOptions)
+                expect(shuffledOptions).toEqual(questionWithoutShufflingOptions.choices)
+            })
 
-    describe('decide response', () => {
```

Comment:
slightly opinionated, but we're mocking load script now so don't need to test this here

(i guess you could test that load script was called but the rest of the tests implicitly test that since they all rely on it so i yeeted)